data/ideas.json

```
@@ -47,4 +47,10 @@
         "level": "basic",
         "tags": "personal-assistant"
     }
```

Please add missing comma sign
```suggestion
    },
```